Add Installation Confirmation Section for Minikube (#17479)
kubernetes/website: content/en/docs/tasks/tools/install-minikube.md

• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)

{{< note >}}

> **Hyper-V**

Reviewer: Maybe remove this line?

Author: Thanks, and sorry for not cleaning this up earlier. Removed with latest push.

{{% /capture %}}

## Confirm Installation

To confirm successful installation of both a hypervisor and `minikube`, you can run the following command to start up a local Kubernetes cluster:

{{< note >}}

> For setting the `--vm-driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--vm-driver` values is available in the [specifying the vm driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).

Reviewer: nit:
```suggestion
For setting the `--vm-driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--vm-driver` values is available in [specifying the VM driver](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
```

{{< /note >}}

```shell
minikube start --vm-driver=<driver_name>
```

Once `minikube start` finishes, run the command below to check the status of the cluster:

```shell
minikube status
```

> `minikube status` should return the following output:

Reviewer: ```suggestion
If your cluster is working OK, the output from `minikube status` is similar to:
```

```
host: Running
kubelet: Running
apiserver: Running
kubeconfig: Configured
```

> The above output from `minikube` confirms `minikube` is working with your chosen hypervisor. To stop your cluster, run the following command:

Reviewer: ```suggestion
After you have confirmed whether Minikube is working with your chosen hypervisor, you can continue to use Minikube or you can stop your cluster. To stop your cluster, run:
```
